### PR TITLE
{lib}[GCCcore/8.3.0] libyaml v0.2.2

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Flamefire** [comment]
@boegel This is for Horovod with TF2 fosscuda, 1 more coming once this is in before horovod

File: easybuild/easyconfigs/l/libyaml/libyaml-0.2.2-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Nils Christian <nils.christian@uni.lu>
+# License::   MIT/GPL
+# $Id$
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'libyaml'
```
Comment (smoors) [inline easybuild/easyconfigs/l/libyaml/libyaml-0.2.2-GCCcore-8.3.0.eb]: @Flamefire it looks like `LibYAML` is the official software name (with capitalization). please change accordingly?

Comment (akesandgren) [reply easybuild/easyconfigs/l/libyaml/libyaml-0.2.2-GCCcore-8.3.0.eb]: Note that the old packaging we have is called libyaml

Comment (Flamefire) [reply easybuild/easyconfigs/l/libyaml/libyaml-0.2.2-GCCcore-8.3.0.eb]: I left it lowercase for consistency. Do we have to rename the folder and filename too? Is it worth it?

Comment (smoors) [reply easybuild/easyconfigs/l/libyaml/libyaml-0.2.2-GCCcore-8.3.0.eb]: @akesandgren ah, I missed that. better to keep the old name then.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node226.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/aee182c50eaf45d91ec59f7f31eab398 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @Flamefire!

**Pull request merged**